keras_nlp/src/models/densenet/densenet_image_classifier.py

```diff
@@ -0,0 +1,124 @@
+# Copyright 2024 The KerasNLP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import keras
+
+from keras_nlp.src.api_export import keras_nlp_export
+from keras_nlp.src.models.densenet.densenet_backbone import DenseNetBackbone
+from keras_nlp.src.models.image_classifier import ImageClassifier
+
+
+@keras_nlp_export("keras_nlp.models.DensekNetImageClassifier")
+class DensekNetImageClassifier(ImageClassifier):
+    """DensekNet image classifier task model.
```

Comment:
This looks copy and pasted directly from the github code viewer which can remove empty newlines. Restore them where applicable.

Reply:
Modified, Thanks